python/inline/conftest.py

```diff
@@ -1028,9 +1071,21 @@ def parse_inline_test(self, node):
         else:
             raise MalformedException("inline test: invalid inline test constructor")
 
-        # "given(a, 1)"
+        # "assume_true(...) or assume_false(...)
         inline_test_call_index = 1
-        for call in inline_test_calls[1:]:
+        for call in inline_test_calls[1:2]:
```

Comment:
It seems to me that this for loop is not necessary. 
call = inline_test_calls[1]